.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish Gem
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+
+      - name: Release Gem
```

Comment:
I don't see where the tags are checked?
shouldn't there be a line `if: contains(github.ref, 'refs/tags/v')` here?

Reply:
or if it is automatic, where i can see that the tag is created automatically when version number is incremented?

Reply:
https://github.com/rubygems/bundler/blob/master/task/release.rake#L133

Reply:
Thank you! :)